src/vivarium_public_health/disease/special_disease.py

```diff
@@ -357,7 +363,9 @@ def adjust_mortality_rate(self, index, rates_df):
         rates_df
 
         """
-        rate = self.excess_mortality_rate(index, skip_post_processor=True)
+        rate = self._get_attribute_pipelines()[self.excess_mortality_rate_pipeline_name](
```

Comment:
This is a problem. Do we need an access pattern to be able to skip the post-processor?